### PR TITLE
Release wf v1

### PR DESCRIPTION
**Pull request by jacobcrawford**

### CODE REVIEW

File: templates/github-workflows/build_release_helm_chart.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      - name: Upload Release Asset
+        id: upload-release-asset 
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}  
+          asset_path: ./${{ env.CHART_NAME }}-${{ steps.previoustag.outputs.tag }}.tgz
+          asset_name: ./${{ env.CHART_NAME }}-${{ steps.previoustag.outputs.tag }}.tgz
+          asset_content_type: application/zip
```
Comment (MartinSchmidt) [inline templates/github-workflows/build_release_helm_chart.yaml:78]: - [ ] Missing empty line at end.

**Review by MartinSchmidt — CHANGES_REQUESTED** [review]
Minor fix, missing empty line at end of .yaml file.

File: templates/github-workflows/build_release_helm_chart.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        run: |
+          sed -i 's/version: v[0-9]\.[0-9]\.[0-9]/version: "${{ steps.previoustag.outputs.tag }}"/g' chart/Chart.yaml
+
+      - name: Commit Helm version
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with: 
+          commit_message: "New helm chart version ${{ steps.previoustag.outputs.tag }}"
+          # As the action is trigger on a tag, no branch exists. Therefore we specify it
+          branch: main
+
```
Comment (MartinSchmidt) [inline templates/github-workflows/build_release_helm_chart.yaml:44]: - [ ] Missing step, move git version tag to new commit.

Maybe update and commit should only run if there were any changes to the file.

File: templates/github-workflows/build_release_helm_chart.yaml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+          
+      - name: Upload Release Asset
+        id: upload-release-asset 
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}  
+          asset_path: ./${{ env.CHART_NAME }}-${{ steps.previoustag.outputs.tag }}.tgz
+          asset_name: ./${{ env.CHART_NAME }}-${{ steps.previoustag.outputs.tag }}.tgz
```
Comment (MartinSchmidt) [inline templates/github-workflows/build_release_helm_chart.yaml:77]: Question: should the name also have a ./ ? since it is the name of the asset?

**Review by MartinSchmidt — CHANGES_REQUESTED** [review]
I left a few comments, on things that should be looked at.

File: templates/github-workflows/build_release_helm_chart.yaml (hunk 3 above)
Comment (jacobcrawford) [reply templates/github-workflows/build_release_helm_chart.yaml]: I am not sure what you mean.

**Review by MartinSchmidt — APPROVED** [review]

File: templates/github-workflows/build_release_helm_chart.yaml (hunk 1 above)
Comment (skovbakke) [reply templates/github-workflows/build_release_helm_chart.yaml:91]: - [ ] Still empty line at the end.

**Review by skovbakke — CHANGES_REQUESTED** [review]
Still no empty line at the end.

**Review by skovbakke — APPROVED** [review]

**Review by MartinSchmidt — APPROVED** [review]

**Pull request merged**